### PR TITLE
Add Cortex doc

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**

### CODE REVIEW

File: docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+uuid: b5968b0a-1206-4d3b-b960-e6b61e7f6ee6
+name: Palo Alto Cortex XDR
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md:2]:
```suggestion
name: Palo Alto Cortex EDR
```

File: docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+type: intake
+
+
+## Overview
+
+Palo Alto Networks Cortex XDR is a comprehensive cybersecurity platform designed to provide advanced threat detection, prevention, and response capabilities. Cortex XDR goes beyond traditional endpoint protection by integrating data from various security sources, including endpoints, networks, and cloud environments, to deliver a holistic view of the organization's security landscape.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md:8]:
```suggestion
Palo Alto Cortex EDR is an advanced Endpoint Detection and Response solution offering real-time threat detection, investigation, and response capabilities, empowering organizations to proactively defend against sophisticated cyber threats across their endpoints.
```

File: docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+uuid: b5968b0a-1206-4d3b-b960-e6b61e7f6ee6
+name: Palo Alto Cortex XDR
+type: intake
+
+
+## Overview
+
+Palo Alto Networks Cortex XDR is a comprehensive cybersecurity platform designed to provide advanced threat detection, prevention, and response capabilities. Cortex XDR goes beyond traditional endpoint protection by integrating data from various security sources, including endpoints, networks, and cloud environments, to deliver a holistic view of the organization's security landscape. 
+
+This setup guide shows how to forward events collected on the Cortex platform to Sekoia.io.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md:10]:
```suggestion
This setup guide shows how to forward EDR events collected on the Palo Alto Cortex XDR platform to Sekoia.io.
```

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for the documentation.
Some improvements suggested.

File: docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+    ![Configuration Page](/assets/operation_center/integration_catalog/cloud_and_saas/paloalto_cortex_edr/configuration.png){: style="max-width:100%; height: 500px;"}
+
+3. In the **Configurations** menu, select **API Keys**, which appears in the **Integrations** section.
+4. In the API Keys window, click **+New Key** button to create a new API key. And the **Generate API Key** window will open.
+
+    ![Api Key Page](/assets/operation_center/integration_catalog/cloud_and_saas/paloalto_cortex_edr/api_key.png){: style="max-width:100%"}
+
+5. Select the **Standard** Security Level.
+6. Select a role for the API key. After selecting the role, click **Generate**.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md:33]: Which role the customer should assign to the API key?

File: docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+    !!! info
+        - You will need also the `Fqdn` which is a unique host and domain name associated with each tenant. When you generate the API Key and Key ID, you are assigned an individual `Fqdn`.
+
+
+## Configure
+
+### Create the intake
+
+1. Go to the [Intake page](https://app.sekoia.io/operations/intakes) and create a new `Palo Alto Cortex EDR` intake.
```
Comment (squioc) [inline docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md:48]:
```suggestion
1. Go to the [Intake page](https://app.sekoia.io/operations/intakes) and create a new `Palo Alto Cortex XDR` intake.
```

**Review by squioc — CHANGES_REQUESTED** [review]

File: docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md (hunk 4 above)
Comment (TOUFIKIzakarya) [reply docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md:33]: Yes they put that we need `Endpoint Management` permission. But as I don't have access to see the api key part in my platform I don't know if it's exactly the same name.

**Review by squioc — APPROVED** [review]

**Pull request merged**